app/src/main/java/de/mvhs/android/zeiterfassung/TimeListActivity.java

```diff
@@ -24,142 +26,158 @@
-            + ",'')<>''";
-    private final static String[] _COLUMNS = {
-            BaseColumns._ID,
-            TimelogContract.Timelog.Columns.START,
-            TimelogContract.Timelog.Columns.END};
-    private final static String[] _UI_COLUMNS = {
-            TimelogContract.Timelog.Columns.START,
-            TimelogContract.Timelog.Columns.END
-    };
-    private final static int[] _ROW_VIEWS = {
-            R.id.StartValue,
-            R.id.EndValue
-    };
-    private ListView _list;
-    private SimpleCursorAdapter _adapter;
-    private final static int _LOADER_ID = 100;
-    private DateFormat _UI_DATE_CONVERTER = DateFormat
-            .getDateTimeInstance(DateFormat.SHORT, DateFormat.SHORT);
-
-    @Override
-    protected void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_time_list);
-
-        // Initialisieren der UI Elemente
-        _list = (ListView) findViewById(R.id.TimelogList);
-
-        // Initialisierung des Adapters
-        _adapter = new SimpleCursorAdapter(
-                getBaseContext(), // Context
-                //android.R.layout.simple_list_item_2, // View für die Darstellung der Daten
-                R.layout.list_row_layout, // Eigenes Layout für die Zeile
-                null, // Daten
-                _UI_COLUMNS, // Spalten für die Darstellung
-                //new int[]{android.R.id.text1,
-                //    android.R.id.text2}, // Views für die Darstellung
-                _ROW_VIEWS, // IDs aus eigenem Layout
-                SimpleCursorAdapter.FLAG_REGISTER_CONTENT_OBSERVER // Auf Änderungen registrieren
-        );
-
-        // Formatieren der Daten aus der Datenbank
-        _adapter.setViewBinder(new SimpleCursorAdapter.ViewBinder() {
-            @Override
-            public boolean setViewValue(View view, Cursor cursor, int columnIndex) {
-                if ((view instanceof TextView) == false) {
-                    return false;
-                }
-
-                // NULL Prüfung aus der Datenbank
-                if (cursor.isNull(columnIndex)) {
-                    ((TextView) view).setText("---");
-                } else {
-                    try {
-                        Date date = TimelogContract.Converter
-                                .DB_DATE_TIME_FORMATTER.parse(
-                                        cursor.getString(columnIndex));
-
-                        ((TextView) view).setText(_UI_DATE_CONVERTER.format(date));
-                    } catch (ParseException e) {
-                        ((TextView) view).setText("PARSE ERROR");
-                    }
-                }
-
-                return true;
-            }
-        });
-
-        // Adapter mit Liste verbinden
-        _list.setAdapter(_adapter);
-    }
+    implements LoaderManager.LoaderCallbacks<Cursor>,
+    ExportProgressDialog.ExportProgressDialogListener {
```

Comment:
Warum implementiert `TimeListActivity` das Interface für das Abbrechen des Exportes, statt der `CsvExporter` Klasse, in der das Dialog erzeugt wird?